components/BattleScreen/Message/LoserMessage.js

```diff
@@ -1,19 +1,23 @@
 import styled from "styled-components";
 
 export default function LoserMessage() {
+	const handleRestart = () => {
+		window.location.reload();
+	};
 	return (
 		<LoserMessageMessageContainer>
 			<LoserTitle>Du hast verloren!</LoserTitle>
+			<RestartButton onClick={handleRestart}>Neustart</RestartButton>
 		</LoserMessageMessageContainer>
 	);
 }
 
 const LoserMessageMessageContainer = styled.section`
-	background-color: #fff;
 	color: #000;
 	width: 100%;
 	height: 100%;
 	display: flex;
+	flex-direction: column;
```

Comment:
`LoserMessageMessageContainer` and `VictoryMessageContainer` are very similar. Please consider to create a generic `MessageContainer` and adjust it using props: [Adapting based on props](https://styled-components.com/docs/basics#adapting-based-on-props)